numpy/testing/pytest_tools/utils.py

```diff
@@ -1619,7 +1619,9 @@ def integer_repr(x):
     """Return the signed-magnitude interpretation of the binary representation of
     x."""
     import numpy as np
-    if x.dtype == np.float32:
+    if x.dtype == np.float16:
```

Comment:
Both this and the version in `numpy/testing/nose_tools/utils.py` seem identical, is there an appropriate place for me to merge this into one, or is seperation preferred?

Reply:
We should probably pull `utils` out into a common file at some point, but for now the two versions need to be treated separately.